src/collection-preferences/content-display/content-display-option.scss

```diff
@@ -21,60 +21,54 @@ $border-radius: awsui.$border-radius-item;
   }
 }
 
-.sortable-item-toggle {
+.content-display-option-toggle {
   /* used in test-utils */
 }
 
-.sortable-item {
-  position: relative;
-}
-
-.sortable-item-placeholder {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  background: awsui.$color-drag-placeholder-hover;
+.content-display-option-content {
+  @include styles.styles-reset;
+  display: flex;
+  align-items: flex-start;
+  padding: awsui.$space-xs awsui.$space-scaled-xs awsui.$space-xs 0;
+  background-color: awsui.$color-background-container-content;
   border-radius: $border-radius;
 }
 
-.sortable-item-content {
+.content-display-option {
+  list-style: none;
+  position: relative;
   border-top: awsui.$border-divider-list-width solid awsui.$color-border-divider-default;
-  display: flex;
-  flex-wrap: nowrap;
-  justify-content: space-between;
-  align-items: flex-start;
-  padding-top: awsui.$space-xs;
-  padding-bottom: awsui.$space-xs;
-  padding-right: 0;
-  &:not(.draggable) {
-    padding-left: awsui.$space-scaled-l;
-  }
-  &.draggable {
-    padding-left: 0;
-    padding-right: awsui.$space-scaled-xs;
-    background-color: awsui.$color-background-container-content;
+  &:not(.placeholder).sorting {
+    @include animated;
     z-index: 1;
-    &.active {
+  }
+  &.placeholder {
+    > .content-display-option-content {
       position: relative;
-      z-index: 2;
-      box-shadow: awsui.$shadow-container-active;
-      border-radius: $border-radius;
-    }
-    &:not(.active).sorting {
-      @include animated;
-    }
-    @include focus-visible {
-      &.active {
-        @include animated;
-        @include styles.focus-highlight(0px);
+      &:after {
+        content: ' ';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background: awsui.$color-drag-placeholder-hover;
+        border-radius: $border-radius;
       }
     }
   }
 }
 
-.sortable-item-label {
-  padding-right: awsui.$space-l;
+.content-display-option-label {
   flex-grow: 1;
+  @include styles.text-wrapping;
+  padding-right: awsui.$space-l;
+}
+
+.drag-overlay {
+  box-shadow: awsui.$shadow-container-active;
+  border-radius: $border-radius;
+  @include focus-visible {
+    @include styles.focus-highlight(0px, $border-radius);
```

Comment:
The second parameter `$border-radius` was previously missing; this fixes the focus ring border radius.